pkg/odo/cli/pipelines/init.go

```diff
@@ -108,7 +115,6 @@ func NewCmdInit(name, fullName string) *cobra.Command {
 	initCmd.Flags().StringVar(&o.gitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
 	initCmd.MarkFlagRequired("gitops-repo-url")
 	initCmd.Flags().StringVar(&o.gitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub webhook secret for GitOps repository")
```

Comment:
Maybe for these, we should mention that we'll automatically generate the secret if it's not provided?

Reply:
Made the changes.